tox.ini

```diff
@@ -20,6 +20,14 @@ commands =
    poetry install -v {env:INSTALL_OPTS}
    poetry run pytest --quiet --cov=generated/nidaqmx --cov-append --cov-report= --junitxml=test_results/system-{envname}.xml {env:PYTEST_OPTS} {posargs}
 
+[testenv:nicaiu]
+setenv =
+   NIDAQMX_C_LIBRARY = nicaiu
+
+[testenv:nicai_utf8]
+setenv =
+   NIDAQMX_C_LIBRARY = nicai_utf8
```

Comment:
This works, but it's less verbose to add these to `[testenv]`'s `setenv = ` section:
```
setenv =
   ...
   nicaiu: NIDAQMX_C_LIBRARY=nicaiu
   nicaiu_utf8: NIDAQMX_C_LIBRARY=nicai_utf8
```

See https://tox.wiki/en/3.1.3/config.html#complex-factor-conditions